doc/building.md

```diff
@@ -1226,6 +1226,42 @@ available using `--with-abi-profile`: arm-vfp-sflt, arm-vfp-hflt, arm-sflt,
 armv5-vfp-sflt, armv6-vfp-hflt. Note that soft-float ABIs are no longer
 properly supported by the JDK.
 
+### Building for RISC-V
+
+The RISC-V community provides a basic [GNU compiler toolchain](https://github.com/riscv-collab/riscv-gnu-toolchain).
+But the [external libraries](#External-Library-Requirements) required by OpenJDK complicate the building process.
+The placeholder `<toolchain-installed-path>` shown below is the path you want to install the toolchain.
```

Comment:
Please keep paragraph text to 80 character width.

"is the path you want" -> "is the path where you want"

Reply:
Fixed.